web/src/pages/index.tsx

```diff
@@ -33,6 +36,29 @@ const actionBoxMockup: CallToActionBoxProps[] = [
   },
 ];
 
+const projectsBoxesData: CallToActionImageProps[] = [
+  {
+    image: 'tabetalt.png',
+    linkText: 'Tabetalt – En nettbutikkleverandør',
+    linkTo: '/projects/tabetalt',
+  },
+  {
+    image: 'oslojazz.png',
+    linkText: 'Oslo Jazz',
+    linkTo: '/projects/oslojazz',
+  },
+  {
+    image: 'lent.png',
+    linkText: 'Lent – Læringsapplikasjon',
+    linkTo: '/projects/lent',
+  },
+  {
+    image: 'indiv.png',
+    linkText: 'Indiv',
+    linkTo: '/projects/indiv',
+  },
+];
```

Comment:
This should probably be handled with `frontmatter`.

Example:

Filename: `web/src/projects/tabetalt.mdx`
```typescript
---
slug: "/projects/tabetalt"
title: "Tabetalt – En nettbutikkleverandør"
---

...
```

Which make it possible for us to do this:

```suggestion
const projectsBoxesData: CallToActionImageProps[] = useStaticQuery(graphql`
	query($slug: String!) {
    markdownRemark(frontmatter: { slug: { eq: $slug } }) {
      frontmatter {
        slug
        title
      }
    }
`);
```

**Note**: What about internalization? re #23 

### References

- [StaticQuery](https://www.gatsbyjs.com/docs/how-to/querying-data/use-static-query/)
- https://www.gatsbyjs.com/docs/how-to/routing/adding-markdown-pages/

Reply:
@cobraz I added mdx support in two last commits but there were a few questions:

Now there are providing components in .mdx files, I can change them to the md-tags, like here
```jsx
// template.js
import { H1Component, H2Component, H3Component } from './components';
...
<MDXProvider
  components={{
    h1: H1Component,
    h2: H2Component,
    h3: H3Component,
    ...
  }}
>
...
```
```markdown
// project.mdx
---
slug: "/project"
title: "title"
image: "image.jpg"
description: "Description"
---
## title2
### title3
text
...
```

But some of the pages (e.g. project pages) have different styles (margins, etc.).
Which is better: provide components as md-tags or use them directly in .mdx files?

Reply:
Its probably a better user experience to provide them as md-tags, right? What do you think?

Reply:
@cobraz  I think yes, but how to style, for example, such symbols/images with md-tags without creating additional styles and components or to create flexible images?
<img width="757" alt="image" src="https://user-images.githubusercontent.com/79641914/112278995-76233780-8c94-11eb-9f31-fb127e6355a6.png">

Reply:
We can have props for md-tags, right?

Reply:
@cobraz yes, but if we create an mdx template for the project model, these props will be the same on each of the pages, right?
```jsx
// templates/project.js
const Heading = ({ children, ...props }) => (
   <BaseHeading {...props}>{children}</BaseHeading>
 );

 export default function ProjectTemplate({ data: { mdx } }) {
   return (
      ...
           <MDXProvider components={{ h1: Heading }}>
             <MDXRenderer>{mdx.body}</MDXRenderer>
           </MDXProvider>
      ...
   );
 }
```
```markdown
// /pages/project1.mdx
---
slug: "/projects/project1"
title: "Title"
...
---
# Heading component text
```

Reply:
Yes, I would probably the `mdx` files in an other location tho'.

e.g. `src/web/content/projects/`?

Reply:
btw, you probably have to use [Gatsby Node API](https://www.gatsbyjs.com/docs/reference/config-files/gatsby-node/) for this btw.